checks/system/win32.py

```diff
@@ -178,6 +183,13 @@ def check(self, agentConfig):
             pct_usable = float(usable) / total
             self.save_sample('system.mem.pct_usable', pct_usable)
 
+        page = psutil.virtual_memory()
+        if page.total is not None:
```

Comment:
Do you know if it's possible to fetch this information from the `Win32_PerfRawData_PerfOS_Memory` WMI class ? See https://github.com/DataDog/dd-agent/blob/master/checks/system/win32.py#L91